content/ideas/hackage-candidate.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Finish the package candidate workflow for Hackage
+---
+
+Hackage candidate packages currently cannot be used directly, and their UI could be improved. We would like to have new packages be uploaded as candidates by default, to improve the vetting process. But this means polishing off candidate functionality. The main issues left to do are tracked [here](https://github.com/haskell/hackage-server/projects/1)
```

Comment:
> We would like to have new packages be uploaded as candidates by default, to improve the vetting process.
Isn't this already the case in modern cabal versions?

Reply:
IIRC if you upload a package from the web UI, it'll be published directly to the main index.

Reply:
I added some text to clear this up.